src/Microsoft.AspNetCore.Razor.Language/SourceSpan.cs

```diff
@@ -30,6 +30,11 @@ public SourceSpan(string filePath, int absoluteIndex, int lineIndex, int charact
             FilePath = filePath;
         }
 
+        public SourceSpan(int absoluteIndex, int lineIndex, int characterIndex, int length) 
```

Comment:
This is equivalent to using the ctor that uses `SourceLocation` but is more convenient when fixing tests.